concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -477,21 +486,116 @@ jobs:
+      - aggregate:
+          - get: pipeline-trigger
+            passed: ['generate-secrets']
+            trigger: true
+          - get: paas-cf
+            passed: ['generate-secrets']
+          - get: cf-secrets
+            passed: ['generate-secrets']
+          - get: bosh-CA
+            passed: ['generate-secrets']
+      - try:
+          task: wait-for-availability-test
+          config:
+            platform: linux
+            inputs:
+              - name: bosh-CA
+              - name: paas-cf
+              - name: cf-secrets
+              - name: pipeline-trigger
+            params:
+              SYSTEM_DNS_ZONE_NAME: {{system_dns_zone_name}}
+              CF_ADMIN: admin
+            image: docker:///governmentpaas/cf-cli
+            run:
+              path: sh
+              args:
+                - -e
+                - -u
+                - -c
+                - |
+                  ./paas-cf/concourse/scripts/import_bosh_ca.sh
+
+                  CF_PASS=$(awk '/uaa_admin_password/ {print $2}' cf-secrets/cf-secrets.yml)
+                  API_ENDPOINT="https://api.${SYSTEM_DNS_ZONE_NAME}"
+                  PIPELINE_TRIGGER_VERSION=$(cat pipeline-trigger/number)
+
+                  echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS} > /dev/null
+                  cf target -o testers -s healthcheck > /dev/null
+
+                  echo "Waiting two minutes for availability test to start:"
+                  for i in $(seq 24); do
+                    if cf logs healthcheck --recent | grep -q "availability-test=${PIPELINE_TRIGGER_VERSION}"; then
+                      echo "Request detected"
+                      break
+                    fi
+                    printf "."
+                    sleep 5
+                  done
+
+
+  - name: availability-tests
+    plan:
+      - aggregate:
+          - get: pipeline-trigger
+            passed: ['generate-secrets']
+            trigger: true
+          - get: paas-cf
+            passed: ['generate-secrets']
+          - get: bosh-CA
+          - get: concourse-manifest
+      - task: run-tests
+        config:
+          platform: linux
+          image: docker:///governmentpaas/cf-acceptance-tests
+          inputs:
+            - name: paas-cf
+            - name: pipeline-trigger
+            - name: concourse-manifest
+            - name: bosh-CA
+          params:
+            SKIP_SSL_VALIDATION: true
+            APPS_DNS_ZONE_NAME: {{apps_dns_zone_name}}
+            SYSTEM_DNS_ZONE_NAME: {{system_dns_zone_name}}
+            CONCOURSE_ATC_USERNAME: admin
+            DEPLOY_ENV: {{deploy_env}}
+            BRANCH: {{branch_name}}
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                ./paas-cf/concourse/scripts/import_bosh_ca.sh
+
+                export CONCOURSE_ATC_PASSWORD=$(awk '/basic_auth_password/ { print $2 }' concourse-manifest/concourse-manifest.yml)
```

Comment:
Can you use `var_from_yaml.rb` here instead of `awk`?

It'll be safer and avoid the problems described in 9b732a66980010a8e4ee468d09d93af6514259e6 and cdcab37f2fe21d4b6e5229fda89de321c39c9b04.